lib/registration.pm

```diff
@@ -679,7 +679,10 @@ sub fill_in_reg_server {
     else {
         send_key "alt-i";
         # Fresh install sles12sp2/3/4/5 shortcut is different with upgrade version
-        if ((is_sle('12-sp2+') && is_sle('<15') && (get_var('UPGRADE') || get_var('ONLINE_MIGRATION'))) || is_sle('>=15')) {
+        # this is a workaround for bug bsc#1141962
```

Comment:
This is caused by different shortcut on patch_sle and scc_register module for smt scenario. So we add check_var(IN_PATCH_SLE) condition.